sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -697,7 +697,7 @@ window.addEventListener("DOMContentLoaded", () => {
         const project = READTHEDOCS_DATA.project;
         const version = READTHEDOCS_DATA.version;
         const language = READTHEDOCS_DATA.language || "en";
-        const api_host = '/_';
+        const api_host = READTHEDOCS_DATA.proxied_api_host || '/_';
```

Comment:
I'd love to standardize this a bit more in some kind of JS client, at some point in the future.